apartment_application_service/settings.py

```diff
@@ -183,6 +186,35 @@
 CORS_ORIGIN_WHITELIST = env.list("CORS_ORIGIN_WHITELIST")
 CORS_ORIGIN_ALLOW_ALL = env.bool("CORS_ORIGIN_ALLOW_ALL")
 
+AUDIT_LOG_FILENAME = env("AUDIT_LOG_FILENAME")
+
+if AUDIT_LOG_FILENAME:
+    if "X" in AUDIT_LOG_FILENAME:
+        import random
+        import re
+        import string
+
+        system_random = random.SystemRandom()
+        char_pool = string.ascii_lowercase + string.digits
+        AUDIT_LOG_FILENAME = re.sub(
+            "X", lambda x: system_random.choice(char_pool), AUDIT_LOG_FILENAME
+        )
+
+    _audit_log_handler = {
+        "level": "INFO",
+        "class": "logging.handlers.RotatingFileHandler",
+        "filename": AUDIT_LOG_FILENAME,
+        "maxBytes": 100_000_000,
+        "backupCount": 1,
+        "delay": True,
+    }
+else:
+    _audit_log_handler = {
+        "level": "INFO",
+        "class": "logging.StreamHandler",
+        "stream": stdout,
```

Comment:
is this extra information as it should be default

Reply:
Good point, `level` and `stream` here should not be needed.